plugins/include/open62541/plugin/log_stdout.h

```diff
@@ -22,6 +48,15 @@ UA_Log_Stdout_log(void *_, UA_LogLevel level, UA_LogCategory category,
 UA_EXPORT void
 UA_Log_Stdout_clear(void *logContext);
 
+/* By default the client and server is configured with UA_Log_Stdout_log
+   These functions sets the context with a max log level */
+UA_EXPORT void
+UA_ClientConfig_setLog_Stdout_withLevel( UA_ClientConfig *config, 
```

Comment:
I'd rather see a "constructor function" that returns the configured UA_Logger structure.
This can be used for the client and server config setup.

```c
UA_Logger UA_Log_Stdout_withLevel(UA_LogLevel minlevel);
```

Remove the _setLog methods.

Reply:
IMHO the `_setLog` is more consistent with how other parts of the client/server is configured. 

You want the users to configure it like this then?
```
conf->logger = UA_Log_Stdout_withLevel(UA_LogLevel minlevel);
```

Reply:
I have added the `UA_Log_Stdout_withLevel` constructor.  
But so far I have kept `UA_ClientConfig_setLog_Stdout_withLevel`, and just changed it to use `UA_Log_Stdout_withLevel`